sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamTest.scala

```diff
@@ -669,55 +658,48 @@ trait StreamTest extends QueryTest with SharedSQLContext with Timeouts {
     }
   }
 
-
-  class QueryStatusCollector extends StreamingQueryListener {
+  /** Collects events from the StreamingQueryListener for testing */
+  class EventCollector extends StreamingQueryListener {
```

Comment:
Ah yes, it was being used in multiple files at some point so I put it here. but not any more. I will put it only in StreamingQueryListener.